apps/verticalhome/js/app.js

```diff
@@ -141,6 +156,38 @@
       this.grid.stop();
     },
 
+    getAppByURL: function(url) {
+      return this.itemStore.getAppByURL(url);
+    },
+
+    getPinAppList: function() {
+      return this.itemStore.getPinAppList();
+    },
+
+    savePinAppItem: function(obj) {
+      this.itemStore.savePinAppItem(obj);
+    },
+
+    savePinApps: function(objs) {
+      this.itemStore.savePinApps(objs);
+    },
+
+    showMoreApps: function() {
+      this.inMoreApps = true;
+      console.log("inMoreApps is : " + this.inMoreApps);
+      document.getElementById("main-screen").setAttribute("hidden","");
+      document.getElementById("more-apps-screen").removeAttribute("hidden");
```

Comment:
Rather than using the hidden attribute here, it is probably better to set and remove a CSS class that will show the more apps screen on top of the main screen. That way you can specify CSS animations for the transition, which is what our UX designers have in mind, I think.